CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 2.0.8 (2023-04-27)
+
+* Change Occurences of sh to bash (#92)
+* Bump Bitnami Chart, ZK Tag, and Update Docs (#93)
```

Comment:
we should probably be more descriptive here that there was a major zk version jump